builder/vmware/common/run_config.go

```diff
@@ -8,6 +8,8 @@ import (
 	"github.com/hashicorp/packer/template/interpolate"
 )
 
+// ~> **Note:** From Packer v1.6.3, remote ESXi builds no longer use VNC to send
+// boot_command keystrokes. Any VNC configuration will be ignored.
```

Comment:
Should we log warnings that users don't need to set these values, if they're set?